app/jobs/create_new_person_job.rb

```diff
@@ -0,0 +1,14 @@
+class CreateNewPersonJob
+  include Sidekiq::Worker
+  sidekiq_options queue:'create_person'
+
+  def perform(person_params)
+    person = Person.new(person_params)
+    if person.save
+      Rails.logger.info(person_params)
+    else
+      Rails.logger.error(action: "create", message: e.message)
```

Comment:
@louy755 are you sure this statement works? I think Rails.logger takes only string.